tests/Integration/Behaviour/Features/Scenario/CmsPage/cms_page_management.feature

```diff
@@ -0,0 +1,82 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s cms_page
+@reset-database-before-feature
```

Comment:
I think this behat test lacks multishop tests - e.g what If I create record in shop1 and shop2. You should test it as well.

Reply:
Great idea, but then let's do it in a 2nd `.feature` file in order to ease the debugging 😉